### PR TITLE
 Changed icons lib to Ikonli FontAwesome 5

### PR DESCRIPTION
**Pull request by skmedix**
https://github.com/aalmiray/ikonli

_Java 8_
**Icons cheat sheet for 4.x:**
https://skmedix.github.io/ikonli-java-8/
**Icons cheat sheet for 4.x:**
https://skmedix.github.io/ikonli-java-8/cheat-sheet-fontawesome5.html

---

_Java 11_
**Usage for 11.x:**
http://aalmiray.github.io/ikonli/
**Icons cheat sheet for 11.x:**
http://aalmiray.github.io/ikonli/cheat-sheet-fontawesome5.html

### CODE REVIEW

**Comment by skmedix** [comment]
Simple comparison:

File: demo/src/main/java/demos/components/ToggleButtonDemo.java [hunk 1]
```diff
@@ -30,11 +28,7 @@ public void start(Stage stage) throws Exception {
         pane.getChildren().add(toggleButton);
 
         JFXToggleNode node = new JFXToggleNode();
-        final GlyphIcon heartIcon = GlyphsBuilder.create(FontAwesomeIconView.class)
-            .glyph(FontAwesomeIcon.HEART)
-            .build();
-        heartIcon.setStyle("-fx-padding: 10");
```
Comment (skmedix) [inline demo/src/main/java/demos/components/ToggleButtonDemo.java]: **Note:** Didn't work anyway.

---

**Before:**
image

**After:**
image

Comment (jfoenixadmin) [reply demo/src/main/java/demos/components/ToggleButtonDemo.java]: it looks better with the padding :), feel free to add it to the PR

**Comment by jfoenixadmin** [comment]
I think this PR will break the main demo, thus I'll postpone the merge a bit until I have some time to fix the demo too.
Thanks @skmedix  for the update.

Regards,

**Comment by skmedix** [comment]
Whoops, my bad. I forgot to commit MainDemo 😅

**Comment by skmedix** [comment]
@jfoenixadmin Should be good now

File: demo/src/main/java/demos/components/ToggleButtonDemo.java (hunk 1 above)
Comment (skmedix) [reply demo/src/main/java/demos/components/ToggleButtonDemo.java]: Changed

**Comment by jfoenixadmin** [comment]
@skmedix great 👍

**Pull request merged**